kagenti/installer/app/components/ui.py

```diff
@@ -21,6 +21,39 @@
 
 def install():
     """Installs the Kagent UI from its deployment YAML."""
+    run_command(
+        [
+            "kubectl",
+            "apply",
+            "-n",
+            "kagenti-system",
+            "-f",
+            str(config.RESOURCES_DIR / "global-environments.yaml"),
+        ],
+        f"Applying global-environments configmap in 'kagenti-system'",
+    )
```

Comment:
Adding the new `global-environments` configmap. Later on, we can also split this up into a configmap and a secret.